csrc/build.rs

```diff
@@ -74,7 +74,10 @@ const KERNEL_FILES: [&str; 65] = [
 
 fn main() -> Result<()> {
     println!("cargo:rerun-if-changed=build.rs");
-    for kernel_file in KERNEL_FILES.iter() {
+    for kernel_file in KERNEL_FILES
+        .iter()
+        .chain(std::iter::once(&"kernels/flash_api.cu"))
```

Comment:
the `flash_api.cu` is already in the `KERNEL_FILES`

Reply:
Good catch !